nautobot_ssot/templates/nautobot_ssot/sync_detail.html

```diff
@@ -58,7 +59,15 @@
                     </tr>
                     <tr>
                         <td>Duration</td>
-                        <td>{{ object.duration | shorter_timedelta }}</td>
+                        <td>
+                            {{ object.duration | shorter_timedelta }} total
+                            <ul>
+                                <li>{{ object.load_time_source | shorter_timedelta }} loading from {{ object.source }}</li>
+                                <li>{{ object.load_time_target | shorter_timedelta }} loading from {{ object.target }}</li>
+                                <li>{{ object.diff_time | shorter_timedelta }} calulating diffs</li>
+                                <li>{{ object.sync_time | shorter_timedelta }} performing sync</li>
```

Comment:
If we're rewriting migrations anyway, my vote would be to also rename `load_time_source` and `load_time_target` to `source_load_time` and `target_load_time` so as to make them more consistent with `diff_time` and `sync_time`.

Reply:
so, from your message, I understand that you agree about rewriting migrations?

Reply:
Yes, I think that should be fine.